models/Article.js

```diff
@@ -11,9 +11,26 @@ export default (sequelize, DataTypes) => {
       },
       title: { type: DataTypes.STRING, allowNull: false },
       description: { type: DataTypes.TEXT, allowNull: false },
-      body: { type: DataTypes.TEXT, allowNull: false }
+      body: { type: DataTypes.TEXT, allowNull: false },
+      readTime: {
+        type: DataTypes.INTEGER,
+        allowNull: true,
+        defaultValue: 0
+      }
     },
-    { tableName: "articles" }
+    {
+      tableName: "articles"
+      // hooks: {
```

Comment:
Why are these comments going to staging?

Reply:
@puritybirir that one was from develop, but now I removed it.